app/models/landmark.rb

```diff
@@ -6,6 +6,8 @@ class Landmark < ActiveRecord::Base
   validates :location, presence: true
   validates :image, presence: true
   validates :description, presence: true
+  validates :user_id, presence: true
```

Comment:
I would prefer to see `validates :user, presence: true`. Ids are something the database takes care of. In Ruby land, we care about objects.